log.go

```diff
@@ -3,33 +3,44 @@ package turnpike
 import (
 	"errors"
 	"fmt"
-	glog "log"
+	"log"
 	"os"
 )
 
 var (
-	logFlags = glog.Ldate | glog.Ltime | glog.Lshortfile
-	log      *glog.Logger
+	loggerFlags = log.Ldate | log.Ltime | log.Lshortfile
+	logger      Logger
```

Comment:
That's true.  This change was mostly to keep my sanity as goimports automatically imports <code>log</code> in other source files that use this global variable.  Will change it back as it will cause less issues when merging from upstream.